CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
I don't think we need this file as we are using setup.py to compile

Reply:
If we're going to have C++ API we do.